src/ansibleschemas/navigator.py

```diff
@@ -0,0 +1,280 @@
+# Used to generate JSON Validations chema for ansible-navigator config files
+# https://github.com/ansible-community/ansible-lint/blob/master/.ansible-lint
```

Comment:
Navigator?

Reply:
That's copy&paste error. I'll fix it

Reply:
Lets wait for @cidrblock on this one, he knows navigator better than anyone else and may be able to spot some schema mistakes.

Reply:
Fixed it with 550100a.